gerrit_changes_to_github.py

```diff
@@ -230,10 +231,28 @@ def changes_apply_filter(args, changes, branches) -> List[str]:
 
     filtered = []
     for change in changes:
-        if [branch for branch, _ in branches if branch in change]:
-            continue
-
-        filtered.append(change)
+        if [branch for branch, _ in branches if branch in change["ref"]]:
+            # Check if this change has a "false positive" comment
+            url = f"https://review.spdk.io/gerrit/changes/{change['id']}/revisions/current/comments/"
+            if (comments := requests.get(url)).status_code != 200:
+                log.info(
+                    f"""Failed to retrieve comments of change 
+                    https://review.spdk.io/gerrit/c/spdk/spdk/+/{change['ref'].split('/')[-2]}. 
+                    HTTP Status Code: {comments.status_code}"""
+                )
+                continue
+            comments = json.loads(comments.text[4:]).get("/PATCHSET_LEVEL", [])
+            if any(re.match(REGEX_FALSE_POSITIVE, comment["message"]) for comment in comments):
```

Comment:
Working on adding a check here to see if the issue number specified in the comment matches an issue at https://github.com/spdk/spdk/issues/{issue_num}, but I wanted to get some feedback first since I'm a Python beginner.

Reply:
The code looks great @anisa-su993 , I have added another comment on how to possibly reduce number of auxiliary API calls, reducing that would be preferable to do before doing "issue-matches" as I expect we might other "flood" the gerrit server.

Regarding style etc. then I'll add an issue describing the need to do a bit of "self-testing" of the ci-infrastructure e.g. running the tools ``black``, ``flake8``, ``isort``, ``mypy`` etc. you can run these manually by doing:

``pre-commit run --all-files``

But this should just be a ci job doing it.